dist/api_data.py

```diff
@@ -983,6 +983,13 @@ def __ge__(self, other):
         size and the default config would extend log files in allocations of
         the maximum log file size.''',
         type='list', choices=['data', 'log']),
+    Config('file_close_sync', 'true', r'''
+        control whether to flush modified tables to storage independent
+        of a global checkpoint when acquiring exclusive access to a table.
+        If set to false, API calls that require exclusive access to tables
+        will return EBUSY if there have been changes made to the table since
+        the last global checkpoint.''',
```

Comment:
The word `close` does not appear in the description at all. If this setting is not related to close, it should be removed from the configuration string name. If it is related to close specifically, it should say that simply. If I were a user, I would not know what API calls require/acquire exclusive access to a table.

Reply:
@sueloverso: Thanks for pointing this out. I have added why `close` in name is relevant. I have also added some text to indicate that this config will only work when logging is disabled.